doc/classes/Camera2D.xml

```diff
@@ -150,6 +150,13 @@
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating" default="false">
 			If [code]true[/code], the camera view rotates with the target.
 		</member>
+		<member name="rotation_smoothing_enabled" type="bool" setter="set_rotation_smoothing_enabled" getter="is_rotation_smoothing_enabled" default="false">
+			If [code]true[/code], the camera's view smoothly rotates to align with its target rotation at [member rotation_smoothing_speed].
+			This property has no effect if [member rotating] is [code]false[/code].
+		</member>
+		<member name="rotation_smoothing_speed" type="float" setter="set_rotation_smoothing_speed" getter="get_rotation_smoothing_speed" default="5.0">
+			The angular speed, in radians per second, of the camera's rotation smoothing effect when [member rotation_smoothing_enabled] is [code]true[/code].
```

Comment:
This is probably incorrect, see https://github.com/godotengine/godot/pull/63773#discussion_r970610491

Reply:
I don't believe so. Rotation doesn't scale with zoom. A half-circle will always be a half circle regardless of distance.

Reply:
> The camera's speed cannot be defined in pixels per second because the number of pixels second travelled will depend on how far the camera is from the target. Camera smoothing uses asymptotic smoothing, not linear interpolation.

There's nothing about zoom here.

Reply:
I understand the meaning now, thank you.